### PR TITLE
Vet clinic database: create animals table

### PR DESCRIPTION
**Pull request by danutibe07**
- [ ] Create tables in SQL.
- [ ] Insert and query data in SQL.
query

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -0,0 +1,9 @@
+CREATE TABLE animals (
+    id int GENERATED ALWAYS AS IDENTITY,
+    name string(100),
```
Comment (PinkMoon25) [inline schema.sql:3]: - [ ] Good job so far, however, I think In PostgreSQL, the string data type is not a valid data type. Instead, you can use VARCHAR or TEXT data types to store character data. please try to fix this

File: data.sql [hunk 2]
```diff
@@ -0,0 +1,4 @@
+INSERT INTO animals (name,date_of_birth,neutered,weight_kg,escape_attempt) VALUES ('Agumon','2020-02-03',true,10.23,0);
+INSERT INTO animals (name,date_of_birth,neutered,weight_kg,escape_attempt) VALUES ('Gabumon','2018-11-15',true,8,0);
+INSERT INTO animals (name,date_of_birth,neutered,weight_kg,escape_attempt) VALUES ('Pikachu','2021-01-07',false,15.04,0);
+INSERT INTO animals (name,date_of_birth,neutered,weight_kg,escape_attempt) VALUES ('Devimon','2017-05-12',true,11,0);
```
Comment (PinkMoon25) [inline data.sql:4]: - [ ] As per the data mentioned in the project requirements, the escape attempt property is 0 only for Agumon, while for the others it should be 2, 1, and 5 please update this data

**Review by pinkmoon25 — CHANGES_REQUESTED** [review]
Hi @danutibe07 ,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
Gitflow :heavy_check_mark: 
Template :heavy_check_mark: 

## Required Changes ♻️

- [x] **Please also fix the MIT link in your README currently it is showing 404 page not found** :+1: 

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: data.sql (hunk 2 above)
Comment (Deepakdanger) [inline data.sql]: - [ ] As requested by the previous reviewer, the escape attempt property is 0 only for Agumon, while for the others it should be Gabumon - 2, Pikachu - 1, and Devimon - 5. please update this data

**Review by Deepakdanger — CHANGES_REQUESTED** [review]
Hi @danutibe07, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mckent05 — APPROVED** [review]
Hi @danutibe07, 

# STATUS: APPROVED ✔️ ✔️ 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations and well done! 🎉 
approved


## Highlights
- All linter checks are passing ✔️ ✔️ 
- Your PR is professional 👏 👏 
- Good job creating your animal's table 👍 👍 

## Optional suggestions
**OPTIONAL*** - To make your repo look more professional, please make sure that your branches have descriptive names that describe the feature of the branch e.g feature/animals-table. Please fix this 👍 👍 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**